README.md

```diff
@@ -53,6 +53,11 @@ The application is configured by setting ```QUAKESTATS_SETTINGS``` environment v
 See example [settings.py](examples/settings.py)
 
 ### Launching the application
+Don't forget install ```quakestats``` package. You can do that after install pip requirements.
```

Comment:
I think it would be much easier to deploy the application using wheel package. 
So I'd probably add small section explaining how to build it.
1. Create dev virtualenv
2. Install requirements
3. use Make bdist to create wheel package

Then deploy that package as you wish e.g.
1. Create deployment virtualenv
2. Install built package inside virtualenv
3. Run any webserver which can work with Flask web application (e.g Twisted)